addOns/custompayloads/src/main/javahelp/org/zaproxy/zap/extension/custompayloads/resources/help/contents/options.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>Options Custom Payloads screen</TITLE>
+</HEAD>
+<BODY>
+<H1>Options Custom Payloads screen</H1>
+<p>
+<strong>Note</strong>: Payload categories which are not listed in the table may be available via the Add button as not all rules which support
+custom payloads have default payloads.
+<p>
+This screen/table allows you to configure <a href="custompayloads.html">Custom Payload</a> options:
+
+<H2>Custom Payloads Table</H2>
+
+<H3>Enabled</H3>
+A checkbox indicating whether or not the particular custom payload is to be used or not.
+
+<H3>Category</H3>
+Indicates the category and by association the scan rule for which the custpm payload value should be used.
+(The categories should be fairly obviously relatable to a scan rule, and are also mentioned in the help entry for the scan rules.)
```

Comment:
I'd remove the first part of the sentence and the parentheses, leaving just the indication that the categories are mentioned in the scan rules' help.